test-cases/gutenberg/group.md

```diff
@@ -0,0 +1,355 @@
+
+##### TC001
+
+### Group block is visible in Inserter menu
+
+-   CLick `AppenderButton ( + )`
+-   Check if the `Group` block is visible
+-   Check if you are able to press and add `Group` block
+
+--------------------------------------------------------------------------------
+
+##### TC002
+
+### Group block after add is selected, empty and renders AppenderButton inside
+
+-   Add a `Group` block 
+-   Check if `Group` block is selected
+-   Check if `Group` block is empty and shows only `AppenderButton`
+
+Expected look:
+ <<<SCREENSHOT>>
+  
+--------------------------------------------------------------------------------
+
+##### TC003
+
+### AppenderButton is render properly
+
+-   Add a `Group` block 
+-   Check if `AppenderButton` renders properly
+-   Check if `AppenderButton` is align with the content
+
+Expected look:
+ <<<SCREENSHOT>>
+
+--------------------------------------------------------------------------------
+
+##### TC004
+
+### Group AppenderButton allows to open Inserter and add new block to group
+
+-   Add a `Group` block 
+-   Check if you are able to open `Inserter` menu by press on `AppenderButton` inside `Group`
+-   Check if the block is added inside the `Group` before the `AppenderButton`
+-   Select a `Group` block once again
+-   Check if you are able to add another block
+-   Check if the block is added inside the `Group` before the `AppenderButton`
+
+--------------------------------------------------------------------------------
+
+##### TC005
+
+### DefaultAppender allows to open Inserter and add block outside the group
+
+-   Add a `Group` block 
+-   Having `Group` block selected press on `DefaultAppender`
+-   Check if the block is added outside after the `Group`
+
+<br />
+
+-   Having `Group` block selected long press on `DefaultAppender`
+-   Select where you want to add block after or before
+-   Check if the block is added outside after/before the `Group`
+
+--------------------------------------------------------------------------------
+
+##### TC006
+
+### DefaultAppender allows to open Inserter and add block inside the group
+
+-   Add a `Group` block 
+-   Add block inside the `Group`
+-   Having newly added block selected press on `DefaultAppender`
+-   Check if the block is added inside the `Group` after the selected block
+
+<br />
+
+-   Having newly added block selected long press on `DefaultAppender`
+-   Select where you want to add block after or before
+-   Check if the block is added inside the `Group` after/before the selected block
+
+--------------------------------------------------------------------------------
+
+##### TC007
+
+### Deep nesting is possible
+
+-   Add a `Group` block 
+-   Nest blocks inside multiple `Group` blocks (at least 3 levels deep)
+-   Check if app do not crashes
+
+--------------------------------------------------------------------------------
+
+##### TC008
+
+### Check if Group placeholder is render on RootList
```

Comment:
It's because placeholder can have different style in RootList and in nested structure and in consequence broke the layout..

As far as i understand the plan r in the future is to remove the placeholder in nested structure to avoid render double dashed border so one of this test becomes to check if we see single dashed border, WDYT ?

Reply:
```suggestion
### Check if Group placeholder is visible for the unselected state
```

Reply:
ok let's keep this one but update the title ☝️